yarn.lock

```diff
@@ -8588,6 +8595,13 @@ lit-element@^2.0.1, lit-element@^2.2.0:
   dependencies:
     lit-html "^1.0.0"
 
+lit-element@^2.2.1:
```

Comment:
We should probably just upgrade our lit-element to 2.2.1 so we are carrying two versions

Reply:
Yes

Reply:
I should be able to make the slider accept lower versions of lit-element too, right?

Reply:
Updated lit-element to 2.2.1 in https://github.com/home-assistant/home-assistant-polymer/pull/3614

Reply:
@thomasloven yes, you can mark it as a peer dependency in your `package.json`, that way you tell people: bring your own lit-element.